services/QuillLMS/config/initializers/rails_admin.rb

```diff
@@ -1,146 +1,432 @@
 # frozen_string_literal: true
 
-Rails.application.config.eager_load do
-  RailsAdmin.config do |config|
-    config.asset_source = :sprockets
+RailsAdmin.config do |config|
+  config.main_app_name = ["Quill Content Overview"]
 
-    config.authorize_with do |controller|
-      current_user = User.find_by(id: session[:user_id])
-      redirect_to main_app.root_path unless current_user&.staff?
+  config.asset_source = :sprockets
+
+  config.authorize_with do |controller|
+    current_user = User.find_by(id: session[:user_id])
+    redirect_to main_app.root_path unless current_user&.staff?
+  end
+
+  config.actions do
+    dashboard do
+      statistics false
     end
+    index                         # mandatory
+    new
+    export
+    bulk_delete
+    show
+    edit
+    delete
+    show_in_app
 
-    config.actions do
-      dashboard do
-        statistics false
-      end
-      index                         # mandatory
-      new
-      export
-      bulk_delete
-      show
-      edit
-      delete
-      show_in_app
-
-      ## With an audit adapter, you can add:
-      # history_index
-      # history_show
-    end
-
-    config.model Concept do
-      configure :concept_results do
-        hide
-        # for list view
-        filterable false
-        searchable false
-      end
+    ## With an audit adapter, you can add:
+    # history_index
+    # history_show
+  end
+
+  config.model "ActivityCategory" do
+    list do
+      exclude_fields :activities, :activity_category_activities
     end
+  end
 
-    config.model User do
-      list do
-        field :password_digest do
-          searchable false
-        end
-        field :name do
-          searchable true
-        end
-        field :email do
-          searchable true
-          # TODO: get this displaying emails
-          # formatted_value do # used in form views
-          #
-          #   bindings[:object].email
-          # end
-          # pretty_value do # used in list view columns and show views, defaults to formatted_value for non-association fields
-          #   bindings[:object].email
-          # end
-        end
-        limited_pagination true
+  config.model 'Activity' do
+    edit do
+      exclude_fields :classroom_units, :classrooms, :unit_activities, :units
+    end
+    list do
+      field :id
+      field :name do
+        searchable true
       end
+      field :classification
+      field :flags
     end
+    show do
+      exclude_fields :teachers
+    end
+  end
 
-    config.model School do
-      list do
-        field :zipcode do
-          searchable true
-        end
-      end
+  config.model 'ActivityClassification' do
```

Comment:
Note, I exclude a lot of fields from the `list` view, so I debated extracting this into a config, e.g. 
```ruby
LIST_VIEW_EXCLUSIONS = {
  'ActivityClassification' => [:activities],
  ...
}
```
But there a good number of models with other configs, I worried that might be too confusing to mix them, so I left it verbose. Open to feedback on this.